src/hooks/useNpgSdk.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import type {
+  FieldId,
+  FieldStatus,
+} from "../features/payment/components/IframeCardForm/types";
+import { getConfigOrThrow } from "../utils/config/config";
+import createBuildConfig from "../utils/buildConfig";
+
+export type SdkBuild = {
+  onChange?: (field: FieldId, fieldStatus: FieldStatus) => void;
+  onReadyForPayment?: () => void;
+  onPaymentComplete?: () => void;
+  onPaymentRedirect?: (urlRedirect: string) => void;
+  onBuildError: () => void;
+};
+
+const noop = () => {
+  // noop
+};
```

Comment:
what's the purpose of this empty function?

Reply:
I'm using this to initialize the sdkReady state as an empty function, so that the returned function can be called without errors even if the sdk build is not ready. I'could change this behavior and make the function throw an error if the sdk is not ready.